src/components/MedicationGroup.js

```diff
@@ -52,6 +58,19 @@ export default function MedicationGroup({
             <img src={editIcon} alt="bearbeiten" height="16px" />
           </IconButton>
         </>
+      ) : (
+        <IconButton
+          right="10px"
+          bottom="10px"
+          onClick={() => handleCheckClick(id)}
+          position="absolute"
+        >
+          {check ? (
```

Comment:
Wouldn't it make more sense to call this isChecked?

Reply:
Good idea! I was thinking about this but couldn't find a good name